cluster/charts/crossplane/values.yaml.tmpl

```
@@ -14,6 +14,10 @@ affinity: {}
 # -- Custom labels to add into metadata
 customLabels: {}
 
+# -- Custom Annotations to add to serviceaccount
+serviceaccount:
```

nitpick: can we use camel case here?
```suggestion
serviceAccount:
```